<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblUserName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>User name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditUser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>226</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblPassword">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>226</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnStartSesion">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>226</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Start sesion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>231</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblAccount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>229</width>
        <height>17</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>17</height>
       </size>
      </property>
      <property name="text">
       <string>Do not have an account?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateAccount">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>229</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Create a new account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
